sdks/python/apache_beam/runners/portability/portable_runner_test.py

```diff
@@ -284,6 +284,7 @@ def create_options(self):
     return options
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="[BEAM-10625]")
```

Comment:
@tvalentyn - - assigned this jira to you.

Reply:
Sure, I added comment about specific environment in the Jira. I will add/extend documentation since https://github.com/apache/beam/pull/12405 is merged.